src/js/components/Calendar/Calendar.js

```diff
@@ -911,6 +915,32 @@ const Calendar = forwardRef(
       </StyledWeek>,
     );
 
+    // While the calendar contains focus and is not animating, set
+    // the focus to the active date.
+    useEffect(() => {
+      if (!animating && active && focus && focusableDateRef?.current)
+        focusableDateRef.current.focus();
+    }, [focusableDateRef, active, focus, animating]);
+
+    useEffect(() => {
+      if (bounds && active) {
+        const normalizedBounds = normalizeInput(bounds);
+        if (!betweenDates(active, normalizedBounds)) {
+          const diff1 = Math.abs(
+            active.getTime() - normalizedBounds[0].getTime(),
+          );
+          const diff2 = Math.abs(
+            active.getTime() - normalizedBounds[1].getTime(),
+          );
+          let closerDate = 1;
+          if (diff1 < diff2) {
+            closerDate = 0;
```

Comment:
are we ever reassigning this? could we clean up?
```const closerDate = diff1 < diff2 ? 0 : 1;```

Reply:
Good call, just pushed up a change to fix this